### PR TITLE
Update teams-standalone-static-tabs-using-spo-sites.md

### PR DESCRIPTION
**Pull request by SubbaReddi**
Adding Troubleshooting guide section. Some developers are missing to add SSO information and content URL in incorrect format.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — COMMENTED** [review]
@scanum - Please request a technical review of this content before I do editing. Thanks.

**Comment by scanum** [comment]
@dariomws

**Comment by dariomws** [comment]
If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=SubbaReddi

File: Teams/teams-standalone-static-tabs-using-spo-sites.md [hunk 1]
```diff
@@ -232,4 +232,31 @@ The JSON file you generate will look something like the one below.
...
+2. SSO(single sign-on) properties missing in manifest
+   Make sure you added following properties in manifest:
+   AAD application ID: 00000003-0000-0ff1-ce00-000000000000
+   Resource Url: {{subdomain}}.sharepoint.com
+      Ex: https://contoso.sharepoint.com
+      Ex2: https://orgtenantname.sharepoint.com
+
+Final generated manifest should be similar as shown here: https://docs.microsoft.com/en-us/MicrosoftTeams/teams-standalone-static-tabs-using-spo-sites#a-sample-manifestjson-file
+
 ```
```
Comment (dariomws) [inline Teams/teams-standalone-static-tabs-using-spo-sites.md:262]:
```suggestion

```

## Troubleshooting guide
The Personal tab with the SharePoint site/page shows a blank screen in the desktop Teams client. In the web client, the tab opens without any issues.

Possible reasons and solutions:
- contentURL and websiteURL may have been set wrong:
	- contentUrl in the manifest must be in following format: {{siteUrl}}/_layouts/15/teamslogon.aspx?SPFX=true&dest={{sitePath}}
   		- Example 1: https://contoso.sharepoint.com/_layouts/15/teamslogon.aspx?SPFX=true&dest=/sites/ContosoHub
   		- Example 2: https://orgtenantname.sharepoint.com/_layouts/15/teamslogon.aspx?SPFX=true&dest=/teams/site2

  	- websiteUrl must be in following format: {{siteUrl}}
  		- Example 1: https://contoso.sharepoint.com/sites/ContosoHub
  		- Example 2: https://orgtenantname.sharepoint.com/teams/site2

- SSO (single sign-on) properties missing in the manifest. Make sure you added the following properties in the manifest:
	- AAD application ID: 00000003-0000-0ff1-ce00-000000000000
	- Resource Url: {{subdomain}}.sharepoint.com
		- Example 1: https://contoso.sharepoint.com
		- Example 2: https://orgtenantname.sharepoint.com
```

**Review by dariomws — CHANGES_REQUESTED** [review]
@SubbaReddi thank you for this pull request.

I'm suggesting some improvements. Could you kindly let me know if sometimes the site loads fine and others with a blank screen when this issue is present, or it always shows a blank screen? (always talking about the desktop client.)

**Comment by dariomws** [comment]
@scanum I think this issue deserves that @get-itips and @tseward look over it.

File: Teams/teams-standalone-static-tabs-using-spo-sites.md (hunk 1 above)
Comment (SubbaReddi) [reply Teams/teams-standalone-static-tabs-using-spo-sites.md:262]: Looks fine to me.
sometimes the site loads fine means If the user opens SharePoint site in one of the browser tab and open teams in other tab, it works. If the user is not opened SharePoint site, it is failing to load.

**Comment by get-itips** [comment]
Looks like this is related to https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6976

File: Teams/teams-standalone-static-tabs-using-spo-sites.md (hunk 1 above)
Comment (get-itips) [reply Teams/teams-standalone-static-tabs-using-spo-sites.md:262]: Looks like your addition is inside the code section (the JSON example file)

**Review by tseward — APPROVED** [review]

File: Teams/teams-standalone-static-tabs-using-spo-sites.md (hunk 1 above)
Comment (tseward) [reply Teams/teams-standalone-static-tabs-using-spo-sites.md:262]: Don't forget to remove any `en-us` from article links.

**Comment by dariomws** [comment]
@SubbaReddi could you kindly commit the changes I'm suggesting? Thank you!

**Review by SubbaReddi — COMMENTED** [review]
Looks fine to me. Let me know, if any other action required from my side

**Comment by dariomws** [comment]
@SubbaReddi you need to click the following button to apply the suggested changes:
image

**Comment by SubbaReddi** [comment]
Done

**Comment by dariomws** [comment]
@SubbaReddi sorry for my mistake. The update was applied wrong. I've created another PR with the suggested changes. Please close this PR as the other should be merged.

Thanks!

**Comment by SubbaReddi** [comment]
Based on Dario comment, closing PR.

**Pull request closed without merging**